dataland-backend/src/main/kotlin/org/dataland/datalandbackend/services/DataOwnersManager.kt

```diff
@@ -171,7 +170,24 @@ class DataOwnersManager(
     }
 
     /**
-     * Method to send an data ownership request if an ownership does not already exist
+     * A method to verify if a company has data owners
+     * @param companyId the ID of the company for which the information should be retrieved
+     */
+    @Transactional(readOnly = true)
+    fun checkCompanyForDataOwnership(companyId: String) {
+        checkIfCompanyIsValid(companyId)
+        val dataOwnersEntity = getDataOwnerFromCompany(companyId)
+        val failException = ResourceNotFoundApiException(
```

Comment:
Man könnte das noch in hinter die IF Abfrage schreiben. Dann wird die failException nu gebaut wenn man sie auhc wirklich braucht

Reply:
ist angepasst.

Reply:
ok